03_python_fastapi_project/main.py

```
@@ -128,6 +161,179 @@ async def delete_product(product_id: int, db: AsyncSession = Depends(get_db)):
     return {"message": "Product deleted successfully"}
 
 
+# Cart endpoints
+@app.get("/cart/{session_id}")
+async def get_cart(session_id: str, db: AsyncSession = Depends(get_db)):
```

Ruff (0.13.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)